UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -255,18 +255,19 @@ public void AffectDamage(float damage, int damageType)
 			}
 			if(attackType == AttackType.Bomb)
 			{
-				TakeBluntDamage(damage);
+				TakeBluntDamage();
 				if(damageToLimb >= DamageThreshold)
 				{
 					DismemberBodyPartWithChance();
 				}
 			}
 
-			if (attackType == AttackType.Fire || attackType == AttackType.Laser || attackType == AttackType.Energy)
+			if(Severity <= DamageSeverity.LightModerate){return;}
```

Comment:
I'm fine with returning in the same line, but in that case remove the brackets.

Reply:
Done 👍